README.md

```diff
@@ -1,98 +1,168 @@
 # Docker Volumes Snapshotter
 
-This repository is useful for create a snapshot of a list of volumes into a
-tar file.
+The snapshotter project was created to fulfill the requirement of saving all the volumes of a Docker container to a tar file, which can be restored at a later time. By properly configuring it, volumes from different containers can be saved in the same tar file.
+
+> The tar file can either already exist or be empty. An empty tar file is not a zero-byte file. To understand this better, refer to the [Initialize `tar` file](#initialize-tar-file) section.
+
+The snapshotter container should run with the same volumes as the container whose data should be saved. One way to achieve this is by running the snapshotter container with the `-volumes-from` flag.
 
 - [Docker Volumes Snapshotter](#docker-volumes-snapshotter)
   - [Build snapshotter image](#build-snapshotter-image)
-  - [Running the snapshotter](#running-the-snapshotter)
+  - [Backup](#backup)
+  - [Restore](#restore)
+  - [Configuration file](#configuration-file)
+    - [Passing the configuration file](#passing-the-configuration-file)
+    - [Configuration format](#configuration-format)
     - [Example](#example)
+  - [Backup file](#backup-file)
+    - [Initialize `tar` file](#initialize-tar-file)
+      - [Using the `backuptar` package](#using-the-backuptar-package)
+      - [Using a CLI command](#using-a-cli-command)
+    - [Passing the backup `tar` file](#passing-the-backup-tar-file)
 
 ![diagram](img/snapshotter-diagram.png)
 
 ## Build snapshotter image
 
-To build the image using this public repository as context you can use the
-following command:
+```yaml
+docker build -t snapshotter:v0.2.0 github.com/NethermindEth/docker-volumes-snapshotter.git#v0.2.0
+```
+
+## Backup
+
+To backup volumes from a Docker container use the `backup` command, bind the volumes, configuration file and the `backup.tar` file.
 
 ```bash
-docker build -t docker-volumes-snapshotter github.com/NethermindEth/docker-volumes-snapshotter.git
+docker run \
+  --rm \
+  --volumes-from <container> \
+  -v $(pwd)/backup.tar:/backup.tar \
+  -v $(pwd)/config.yml:/config.yml \
+  eigenlayer-snapshotter:v0.2.0 backup
 ```
```

Comment:
Put a link or reference to the configuration file section, explain what is the `backup.tar` file, explain what value should `<container>` be.

Reply:
also replace `bind volumes` with `bind-mount volumes` in the entire Readme, as that is the original name from the docker docs (would be useful to refer a link to the docker docs for bind-mount volumes in the first mention to bind-mount volumes)

Reply:
Done https://github.com/NethermindEth/docker-volumes-snapshotter/pull/1/commits/89ee98b0715b976c409e5f52a1a8b5d9ae959511